### ISSUE TITLE
Pell WYSIWYG editor color not accessible

### ISSUE THREAD

**Issue by FranceJ** [ifmeorg/ifme#1898]
# Description
The active colour for the Pell editor options is not an extreme enough colour change. On the current background I actually needed to squint very hard at the screen in order to tell that the italics button had been engaged successfully. For accessibility purposes, it should be obvious that colour has changed between active and inactive.

In this case, the switch from white to the current green/yellow colour only has a contrast ratio of 1.34:1. For text displayed on a background for bolded text it is required to be at least 3. I know the implementation is different to text displayed on a background here but I think the same basic principle holds. The contrast must be strong enough between the two states so that folks with accessibility issues can engage with the controls properly.

# Reproduction Steps
Go to `/users/edit` and scroll down to the About section textarea. Toggle the Pell editor styling options on and off and observe the difference in colour.

# Screenshots
Inactive: 
![image](https://user-images.githubusercontent.com/17236079/95643857-06bbe180-0a80-11eb-93e5-16dc70c6209b.png)

Active:
![image](https://user-images.githubusercontent.com/17236079/95643853-fa378900-0a7f-11eb-9192-913ea5e370c3.png)

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by julianguyen** [comment]
Thanks so much for catching this!

**Comment by andrearosr** [comment]
Hello! I would like to collaborate with this issue, however, I've struggled to find a color in your design guide that passed both the contrast between the background color (mulberry #6d0839) **and** with white. The only colors from the design guide that pass the contrast check with a mulberry background (light-grey, key-lime (current) and cornflower) all have very low contrast with white.
Do you have any suggestions for this? I can suggest another way of indicating that the button has been activated, such as a box shadow on the selected buttons:
![image](https://user-images.githubusercontent.com/12634807/96390073-70bb4180-1178-11eb-869f-88ce46c5046f.png)

**Comment by PochecoPachico** [comment]
Hello! I'm also collaborate with this issue. It is great idea, however, I thought that it might be difficult to distinguish between on and off in underscore button. So, I'd like to suggest to change default (off) button color to grey like screenshots I have attached. 
I'm sorry if I violate the style rule. I hope this helps you.
Thank you!

Inactive:
![Screen Shot 2020-12-13 at 11 15 18](https://user-images.githubusercontent.com/12394598/102001345-7f674480-3d34-11eb-8089-1f7fc69a105d.png)


Active: (Bold and Italic buttons)
![Screen Shot 2020-12-13 at 11 14 43](https://user-images.githubusercontent.com/12394598/102001340-6eb6ce80-3d34-11eb-9093-6eac5fdee4bc.png)

**Comment by julianguyen** [comment]
Hi @PochecoPachico! Sorry for the late reply! 

So I don't think grey works here because its contrast with the mulberry colour doesn't meet accessibility standards. You can use colour contast checkers like [this](https://webaim.org/resources/contrastchecker/) verify.

I think the better solution is finding a different colour than `key-lime` so that there is more contrast between that and the white.

We could also use a background colour on the active icons.

**Comment by YusufSuleman** [comment]
Maybe we can change the size of the Icon.

**Comment by julianguyen** [comment]
Oooh @YusufSuleman that's a great suggestion!

**Comment by PochecoPachico** [comment]
@YusufSuleman Thank you for great suggestion! 
I simply implemented your idea. I've attached images.
I want to make sure there are no discrepancies.

Thank you!

Inactive:
<img width="984" alt="Screen Shot 2021-01-09 at 15 11 01" src="https://user-images.githubusercontent.com/12394598/104084726-f4269380-528c-11eb-8316-8fdc5d65a806.png">

Active:
<img width="981" alt="Screen Shot 2021-01-09 at 15 13 35" src="https://user-images.githubusercontent.com/12394598/104084759-42d42d80-528d-11eb-8d86-0e464bbc886e.png">

**Comment by MetaMan13** [comment]
Hello wonderful people of the internet. I am a junior level developer and I was searching for some open source projects I can contribute to. I found your project and I am very interested in contributing to this project. Hereby I ask the contributors of this project if it is okay that I contribute to this too. I would also be very thankful to anyone willing to clue me in to this particular project since this is my first time ever contributing to open source and working with a team. Thanks in advance and have a nice day :D

**Comment by bennpham** [comment]
@MetaMan13 I think you can join the Slack channel and if there's no one assigned to this issue, feel free to assign yourself and take it.

**Comment by YusufSuleman** [comment]
Will this be ever finished. I have completed the W3Schools CSS Tutorial and can implement the size change.

**Comment by julianguyen** [comment]
Hey @YusufSuleman! That's awesome you did that. We would love for you to pick this up!

**Comment by Tejesh17** [comment]
Hey, do you guys still need help on this?

**Comment by julianguyen** [comment]
@YusufSuleman how is this going?

**Comment by Flabberdapperdan** [comment]
Hi there! I'm looking to make my first open-source contribution and this seems well within my capabilities. So, would love to help out if there are problems with implementing this.

**Comment by adang48** [comment]
@YusufSuleman are you still working on this?
